### PR TITLE
Seelog is now integrated into MergeForward

### PR DESCRIPTION
**Pull request by Tanz0rz**
The test file looks pretty gross on the merge view, but the gist of it is I made a blanket Convey statement that sets up the seelog logger. Every test is nested within that. In order to do this, I left the test object out of every sub-convey statement call since the setup step had the reference to in.

### CODE REVIEW

File: mergeforward.go [hunk 1]
```diff
@@ -22,42 +23,77 @@ var outputdir = flag.String("outputdir", ".", "The output directory of the log f
...
+	currentDirectory, err := filepath.Abs(filepath.Dir(os.Args[0]))
+	if err != nil {
+		log.Fatal(err, "- This error happened while automatically detecting the current directory of mergeforward")
+	}
+
+	fmt.Print("Printing current directory: ")
+	fmt.Println(currentDirectory)
+
+	logger, err := seelog.LoggerFromConfigAsString(
+		"<seelog type=\"asynctimer\" asyncinterval=\"1000000\">" +
```
Comment (weberr13) [inline mergeforward.go]: try using `instead of escaping all these characters.`` means "literal string" and it cleans this sort of thing up immensely.

**Comment by weberr13** [comment]
:+1: other than using `` around your literal strings.

File: mergeforward.go (hunk 1 above)
Comment (Tanz0rz) [inline mergeforward.go]: Well... that would have saved me a tedious 10 minutes of backslashing. Good to know.

**Comment by vrecan** [comment]
should we merge this?

**Pull request merged**